contrib/test_jenkins.sh

```diff
@@ -979,12 +979,14 @@ test_jucx() {
                         fi
                         echo "Running standalone benchamrk on $iface"
 
-                        java -cp bindings/java/src/main/native/build-java/jucx-*.jar \
+                        java -XX:ErrorFile=/scrap/core_files/java_$BUILD_NUMBER_pid%p.log \
```

Comment:
can we avoid system dependency?
maybe read sysctl kernel.core_pattern?
BTW which kind of file is it? is it a textual log file, or core dump file?

Reply:
need to pack the java log file as Jenkins build artifact